src/framework/preferences/css/stylus/SeparatedPanelPrefsEditorFrame.styl

```diff
@@ -53,13 +54,17 @@ html {
             box-shadow: 2px 2px #7a766d;
             display: inline-block;
             padding: 1em;
-            min-height: 9.6em;
-            width: 25em;
+            min-width: 25em;
             background-color: #fff;
-            margin-right: 10px;
             white-space: normal;
-            vertical-align: middle;
+            display: table-cell;
         }
+        
+        // override inline label style for panel description 
+        .fl-prefsEditor-display-block {
```

Comment:
instead of making this special class, i wonder if we just add styling for an `fl-prefsEditor-toc-descr` class. e.g.

```css
label .fl-prefsEditor-toc-descr {
    display: block; 
    margin-bottom: 1em;
}
```